### DISCUSSION TITLE
Can anyone fork this as a SaaS ERP?

### DISCUSSION THREAD

**Discussion by davidjeba** [abpframework/abp#9233] [Ideas]
I can provide inputs to create a powerful ERPframework like Odoo, ErpNext etc

Anyone love to work on this?

**Comment by XuJin186** [comment]
This is a very good proposal, what are you planning?

**Comment by davidjeba** [comment]
A clone of ERPNext / odoo on dotnet
Let the developers build their apps for its marketplace

**Reply by XuJin186** [reply]
Whether the workload is too big?

**Reply by davidjeba** [reply]
can you create UI like http://xvelopers.com/demos/html/paper-panel/index.html
on abp.io ?

with tabbed interface?

**Comment by ultimatejasper** [comment]
Very good ideas. We should  develop module by module.

**Comment by davidjeba** [comment]
Can we build multi vendor, omni channel ERP Framework?

i can create requirements and align it to the milestones
need architect [to build framework with tight integration with cloudflare
workers and routing]
need developers [to create milestones and work on it]
need UI designers [to work on designs for multiple devices]
once the product comes to a shape, we can plan other resources.

**Comment by chems** [comment]
Hello,
I have the same challenge an ERP for education.
Any recommandation !!! Thx 😉

**Reply by davidjeba** [reply]
I too wish, abpframework to create official solutions like https://erpnext.com/open-source-education

**Reply by chems** [reply]
https://github.com/dinhduongha/bamboomodules
